lms/views/predicates/_lti_launch.py

```diff
@@ -42,7 +44,7 @@ def __call__(self, context, request):
         return has_document_url == self.value
 
 
-class BlackboardCopied(Base):
+class _CourseCopied(Base, ABC):  # pragma: nocover
```

Comment:
Why is the `# pragma: nocover` needed here? Is that because coverage reports will otherwise complain about the abstract methods not being executed?